SteamList/Views/FavoritesTableViewCell.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  FavoritesTableViewCell.swift
+//  SteamList
+//
+//  Created by Macbook on 05/01/2023.
+//
+
+import UIKit
+
+class FavoritesTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var price: UILabel!
+    @IBOutlet weak var title: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        selectionStyle = .none
```

Comment:
You can safely move selection style to xib file since it is already responsible for layout

Reply:
done